frontend/js/src/explore/artist-similarity/artist-panel/utils.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+
+export function formatReleaseDate(releaseDate: string) {
+  return new Intl.DateTimeFormat("en-US", {
+    month: "short",
+    day: "numeric",
+  })
+    .formatToParts(new Date(Date.parse(releaseDate)))
+    .reverse()
+    .map((date_parts) => date_parts.value)
+    .join("");
+}
+
+// Originally from https://usehooks-ts.com/react-hook/use-media-query
+export function useMediaQuery(queryStr: string) {
```

Comment:
Seems unused, can be removed?

Reply:
I used the same file required for ReleaseCard so should I make any changes to it